FreeRTOS/iot-reference-arm-corstone3xx
# Simplify TF-M Build Configuration Options

Simplify TF-M Build Configuration Options 

Description
-----------
To simplify the build options for TF-M in CMake, rather than declaring separate variables, pass these as part of `TFM_CMAKE_ARGS` to TF-M.

Test Steps
-----------

Checklist:
----------


- [N/A] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
